docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+api:
+  build: .
+  ports:
+    - '8080'
```

Comment:
Change it to `8080:8080`, so that the exposed port is the same as the one that would have been exposed by running the jar directly.